packages/cf-component-card/src/CardToolbarLink.js

```diff
@@ -36,25 +38,33 @@ class CardToolbarLink extends React.Component {
   }
 
   render() {
+    const { id, className, to, children } = this.props;
     return (
       <Link
         role="tab"
-        id={this.props.id}
-        className={this.props.className}
+        id={id}
+        className={className}
+        to={to}
```

Comment:
The role should be "link" if opens a link rather than a drawer

Reply:
fixed, thank you.